doc/crypto/CTLOG_STORE_new.pod

```diff
@@ -0,0 +1,75 @@
+=pod
+
+=head1 NAME
+
+CTLOG_STORE_new, CTLOG_STORE_free,
+CTLOG_STORE_load_default_file, CTLOG_STORE_load_file -
+Create and populate a Certificate Transparency log list
+
+=head1 SYNOPSIS
+
+ #include <openssl/ct.h>
+
+ CTLOG_STORE *CTLOG_STORE_new(void);
+ void CTLOG_STORE_free(CTLOG_STORE *store);
+
+ int CTLOG_STORE_load_default_file(CTLOG_STORE *store);
+ int CTLOG_STORE_load_file(CTLOG_STORE *store, const char *file);
+
+=head1 DESCRIPTION
+
+A CTLOG_STORE is a container for a list of CTLOGs (Certificate Transparency
+logs). The list can be loaded from one or more files and then searched by LogID
+(see RFC 6962, Section 3.2, for the definition of a LogID).
+
+B<CTLOG_STORE_new>() creates an empty list of CT logs. This is then populated
+by B<CTLOG_STORE_load_default_file>() or B<CTLOG_STORE_load_file>().
+B<CTLOG_STORE_load_default_file>() loads from the default file, which is named
+"ct_log_list.cnf" in the OpenSSL install directory. This can be overridden using
```

Comment:
"the OpenSSL install directory" isn't entirely accurate I think. It would be more correct to talk about OPENSSLDIR (although I'm not sure we really define that anywhere in the docs).